### PR TITLE
thrile/5: Added liberate example code from RAK

### PR DESCRIPTION
**Pull request by thrile**

### CODE REVIEW

**Review by ali-abdullah — APPROVED** [review]
No idea what's going on, but LGTM

File: .vscode/cirkopt.code-workspace [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (jameswolfeoliver) [inline .vscode/cirkopt.code-workspace:1]: We should maybe add a `.vscode/*` to the `.gitignore` and not commit these files

File: liberate/cells.tcl [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Comment
+set cells {
+  AND2X1
+  INVX1
+  XOR2X1
+}
+# Overrides
+set cells { AND2X1 INVX1 }
```
Comment (jameswolfeoliver) [inline liberate/cells.tcl:8]: Should there be only one `set cells { ... }` block? I assume the second overwrites the first?

File: liberate/models/ictfile [hunk 3]
```diff
@@ -0,0 +1,1040 @@
+process "GPDK45" {
```
Comment (jameswolfeoliver) [inline liberate/models/ictfile:1]: No idea what this is but I'm assuming it's an "input" file.

If it is an "output" file, maybe we should add it to the .gitignore. Outputs should be ignored to keep diffs more readable

**Review by jameswolfeoliver — APPROVED** [review]
From what I understand this looks good

**Pull request merged**